packages/management/framework/bundle/config/pacakgeinstalls/addons-manager-pi.yaml

```diff
@@ -5,17 +5,22 @@
 apiVersion: packaging.carvel.dev/v1alpha1
 kind: PackageInstall
 metadata:
-  name: tanzu-addons-manager
+  name: tanzu-addons-manager-v2
```

Comment:
**question**: Why are we changing the name of the package install ? For upgrade of management-cluster I think we will have to make sure the following:

1. Delete addon secret for addons-manager so that older addons-controller doesnt reconcile it again.
2. Make sure this PackageInstall can take over the existing addons manager PackageInstall.

@vijaykatam

Reply:
Yes. I will update the naming. I was testing this because I already had a `tanzu-addons-manager` on my cluster and forgot to update this back to original name. I will update this back to original name.

Reply:
> Delete addon secret for addons-manager so that older addons-controller doesnt reconcile it again.

Can you please explain more on this? Is this about the `addons-manager` running on old management cluster? Is the `addons-manager` deployed using PackageInstall on old management cluster as well?

Reply:
> Delete addon secret for addons-manager so that older addons-controller doesnt reconcile it again.

I have filed separate issue to track this work: https://github.com/vmware-tanzu/tanzu-framework/issues/2147

Reply:
The temporary rename is because bolt cannot support multiple versions of the same package in the release graph. We will drop v2 prior to release that uses this.